data/data/bootstrap/baremetal/systemd/units/ironic.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Baremetal Deployment Ironic Services
-Wants=network-online.target crio.service
-After=network-online.target crio.service
+Wants=network-online.target crio.service release-image.service
```

Comment:
> https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=

If the release-image fails transiently, this service will still start.. right? that means the podman inspect will keep failing.. creating noise


and we can't use Requires because of https://github.com/systemd/systemd/issues/1312

Reply:
Can you have a look at the updated approach? release-image is back to being a oneshot, but the retry logic is in the script. It will not complete until the image successfully downloads.  I do use `Requires` in this case, because release-image won't fail, it just keeps retrying.

release-image starts:

```
Aug 26 11:58:48 localhost systemd[1]: Starting Download the OpenShift Release Image...
Aug 26 11:58:48 localhost release-image-download.sh[1699]: Pulling registry.svc.ci.openshift.org/ocp/release:4.2...
Aug 26 11:58:48 localhost podman[1704]: 2019-08-26 11:58:48.997851388 +0000 UTC m=+0.232197590 system refresh
Aug 26 11:58:59 localhost podman[1704]: 2019-08-26 11:58:59.126738877 +0000 UTC m=+10.361085080 image pull  
Aug 26 11:58:59 localhost release-image-download.sh[1699]: c7a857fb2b0c9c7fc6a17efd848213dc1c1de874515e48286bbf907bccbf8055
Aug 26 11:58:59 localhost systemd[1]: Started Download the OpenShift Release Image.
```

And then immediately after finishing, bootkube starts:

```
Aug 26 11:58:59 localhost systemd[1]: Started Bootstrap a Kubernetes cluster.
```